src/KeyringController.test.ts

```diff
@@ -382,16 +345,14 @@ describe('KeyringController', () => {
       it('should add an `encryptionSalt` to the `memStore` when a new vault is created', async () => {
         const keyringController = await initializeKeyringController({
           password: PASSWORD,
+          constructorOptions: {
+            cacheEncryptionKey: true,
+          },
         });
-        keyringController.cacheEncryptionKey = true;
 
-        const initialMemStore = keyringController.memStore.getState();
         await keyringController.createNewVaultAndKeychain(PASSWORD);
-        const finalMemStore = keyringController.memStore.getState();
-
-        expect(initialMemStore.encryptionKey).toBeUndefined();
-        expect(initialMemStore.encryptionSalt).toBeUndefined();
```

Comment:
These are not applicable anymore, as `cacheEncryptionKey` is set to true on construction (before the first `createNewVaultAndKeychain > persistAllKeyrings` is called), so `encryptionKey` and `encryptionSalt` will be already defined